src/server/ApiRoutes/Hold.js

```diff
@@ -431,17 +407,18 @@ function createHoldRequestServer(req, res, pickedUpBibId = '', pickedUpItemId =
     docDeliveryData,
     itemSource,
     (response) => {
-      console.log('Hold Request Id:', response.data.data.id);
-      console.log('Job Id:', response.data.data.jobId);
+      const data = JSON.parse(response).data;
+      console.log('data', data);
```

Comment:
Right, I only intentionally updated the first file the app hits `server.js` to test that the logs correctly go to cloudwatch. The whole app will need an update regarding logging to be consistent, but yes, it'll be a different PR.